src/targets/gpu/fuse_ops.cpp

```diff
@@ -152,6 +154,12 @@ struct fusion
     }
 };
 
+std::unordered_set<std::string> get_supported_archs()
```

Comment:
This should return a const ref: `const std::unordered_set<std::string>&`.

Reply:
Fixed